### ISSUE TITLE
Customize keycloak login theme

### ISSUE THREAD

**Issue by rokroskar** [SwissDataScienceCenter/renku#196]
reuse https://github.com/SwissDataScienceCenter/docker-keycloak-postgres-proxy/pull/1 to customize the look and feel of the login page -- once #187 is merged, this will be the first page users see.

**Comment by leafty** [comment]
Bumping this. Should now be doable in a reasonable way, see: https://github.com/helm/charts/tree/master/stable/keycloak#providing-a-custom-theme.

**Comment by rokroskar** [comment]
great, I hadn't seen that doc before. But in the original theme setup from last year there was quite a lot of other magic that took place -- I guess with the new chart version this is no longer needed?
